### PR TITLE
Responsive pages

### PR DESCRIPTION
**Pull request by gabe2driggs**

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -64,6 +65,12 @@ const App = (props: AppProps) => {
             <CartContextProvider>
               <AppContextProvider>
                 <ProfileContextProvider>
+                  <Head>
```
Comment (adikancherla) [inline pages/_app.tsx:68]: move this to _document.tsx

File: pages/collection/[name]/index.tsx [hunk 2]
```diff
@@ -68,10 +70,13 @@ export default function ItemsPage(props: CollectionDashboardProps) {
     CollectionPageTabs.Analytics.toString()
   ];
 
+  const types = ['Collections', 'Graph'];
```
Comment (adikancherla) [inline pages/collection/[name]/index.tsx]: use ['Items', 'Charts']; rename the variable to 'viewTypes'

File: pages/collection/[name]/index.tsx [hunk 3]
```diff
@@ -68,10 +70,13 @@ export default function ItemsPage(props: CollectionDashboardProps) {
...
+
   const { cartType, setCartType } = useCartContext();
   const [numSweep, setNumSweep] = useState('');
   const [bidBelowPct, setBidBelowPct] = useState('');
   const [customSweep, setCustomSweep] = useState('');
+  const [type, setType] = useState(types[0]);
```
Comment (adikancherla) [inline pages/collection/[name]/index.tsx]: use a more descriptive name - see above comment

File: src/components/common/centered-content.tsx [hunk 4]
```diff
@@ -10,7 +10,7 @@ export const CenteredContent = ({ children }: Props) => {
 
 export const CenterFixed = ({ children }: Props) => {
   return (
-    <div className="top-0 left-0 right-0 bottom-0 fixed flex justify-center items-center pointer-events-none">
+    <div className="top-0 left-16 right-0 bottom-0 fixed flex justify-center items-center pointer-events-none">
```
Comment (adikancherla) [inline src/components/common/centered-content.tsx:13]: why left-16? did you check if this affects all comps that use this correctly?

Comment (gabe2driggs) [reply src/components/common/centered-content.tsx:13]: Yes, because main content starts from `left-16` after left side bar.

**Pull request merged**